web/src/app/shared/dialog/list/dialog-list.component.ts

```diff
@@ -18,13 +20,16 @@ export class DialogListComponent {
         private repositoryService: RepositoryService,
         public dialogRef: MatDialogRef<DialogListComponent>,
         @Inject(MAT_DIALOG_DATA) public data: { project: ProjectModel, repositories: RepositoriesModel },
+        private spinnerService: SpinnerService
     ) {
     }
 
     // @TODO: move out, otherwise can not be reused
     public refresh(): void {
         this.repositorySubscription = this.repositoryService
-            .findAll(true)
+            .findAll(true).pipe(
```

Comment:
`pipe` should go on a new line, but not a big deal

Reply:
I will fix it